From: Theodore Dubois <tblodt@icloud.com>
Date: Mon, 20 Jan 2020 14:10:53 -0800
Subject: tcp: remove redundant assigment to snd_cwnd
Patch-mainline: v5.5
Git-commit: bfe02b9f9476bc8784ebb0f78fa244ad15bb15ea
References: bsc#1154353

Not sure how this got in here. git blame says the second assignment was
added in 3a9a57f6, but that commit also removed the first assignment.

Signed-off-by: Theodore Dubois <tblodt@icloud.com>
Signed-off-by: Eric Dumazet <edumazet@google.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 net/ipv4/tcp.c |    1 -
 1 file changed, 1 deletion(-)

--- a/net/ipv4/tcp.c
+++ b/net/ipv4/tcp.c
@@ -2609,7 +2609,6 @@ int tcp_disconnect(struct sock *sk, int
 	WRITE_ONCE(tp->write_seq, seq);
 
 	icsk->icsk_backoff = 0;
-	tp->snd_cwnd = 2;
 	icsk->icsk_probes_out = 0;
 	icsk->icsk_rto = TCP_TIMEOUT_INIT;
 	tp->snd_ssthresh = TCP_INFINITE_SSTHRESH;
